net/tlsdial/blockblame/blockblame.go

```diff
@@ -0,0 +1,89 @@
+// Package blockblame blames specific firewall manufacturers for blocking Tailscale,
+// by analyzing the SSL certificate presented when attempting to connect to a remote
+// server.
+package blockblame
+
+import (
+	"crypto/x509"
+	"strings"
+)
+
+// Manufacturer represents a firewall manufacturer that may be blocking Tailscale.
+type Manufacturer struct {
+	// Name of the firewall manufacturer, e.g. "Fortinet"
+	CompanyName string
+	// List of indicators that can be used to identify the manufacturer
+	Indicators []Indicator
+}
+
+// Indicator represents a specific piece of information within a SSL certificate that can be
+// used to identify the manufacturer of a firewall.
+type Indicator struct {
+	IssuerName string // never empty, matches if Issuer contains this
+	EmailAddr  string // may be empty, matches if EmailAddresses contains this
+}
+
+var Manufacturers = []Manufacturer{
```

Comment:
let's use `*Manufacturer` here since that's what you return below

Reply:
Done.